src/auth.ts

```diff
@@ -66,21 +66,19 @@ async function authorize(useLocalhost: boolean, writeToOwnKey: boolean) {
 /**
  * Loads the Apps Script API credentials for the CLI.
  * Required before every API call.
- * @param {Function} cb The callback
- * @param {boolean} isLocal If we should load local API credentials for this clasp project.
  */
-export function getAPICredentials(cb: (rc: ClaspSettings | void) => void) {
-    DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
+export async function getAPICredentials() {
```

Comment:
Now that we don't care about the Promise's return values (we still care about the Promise though), we can rename this to `loadAPICredentials`.